pyvista/utilities/common.py

```diff
@@ -23,9 +23,11 @@ def _coerce_pointslike_arg(
 
     Returns
     -------
-    np.ndarray
+    points : np.ndarray
         Size (n, 3) array.
-
+    singular : bool
+        Whether the input array was a single point. This excludes an array with
+        length 1.
```

Comment:
This is definitely subjective, but I'd prefer to be exact about the distinction, because it should be unambiguous. Something like
```suggestion
        Whether the input was a single point. This excludes a 2d array-like with
        shape ``(1, 3)``.
```

Or may be go even more direct and just say "Whether the input was a single point in an array-like with shape `(3,)`".

Reply:
Took your later suggestion.  It is the most direct.  This is also a private helper, so direct is best IMO.